### DISCUSSION TITLE
AssertionError: Could not find PixArt checkpoint at epoch_1_step_8000.pth

### DISCUSSION THREAD

**Discussion by YANG199301** [PixArt-alpha/PixArt-alpha#72] [Q&A] [answered]
test_pixart_control.sh

 assert os.path.isfile(model_name), f'Could not find PixArt checkpoint at {model_name}'         
AssertionError: Could not find PixArt checkpoint at epoch_1_step_8000.pth

**Comment by lawrence-cj** [comment]
This sh file is mis-uploaded. We are preparing for the ckpt. It will be released ASAP.

**Reply by YANG199301** [reply]
Thank you very much, I am looking forward to it.

**Reply by lawrence-cj** [reply, marked as answer]
All the ControlNet-related checkpoints are released, check out the README.md for more info. Thx
